packages/verification-app/.env

```diff
@@ -1,9 +1,9 @@
 VUE_APP_I18N_LOCALE=en
 VUE_APP_I18N_FALLBACK_LOCALE=en
-VUE_APP_CLAIM_FF=
-VUE_APP_PROVIDER_URL=
-VUE_APP_LEGACY_CONTRACT_ADDRESS=
-VUE_APP_LEGACY_CONTRACT_FALLBACK_ADDRESSES=
-VUE_APP_CLAIM_CONTRACT_ADDRESS=
-VUE_APP_ACCEPTED_ISSUER_KEY=
-VUE_APP_CERTIFACTION_API_URL=
+VUE_APP_CERTIFACTION_API_URL=https://api.dev.testnet.certifaction.io/
+VUE_APP_PROVIDER_URL=https://ropsten.infura.io/v3/4876e0df8d31475799c8239ba2538c4c
+VUE_APP_LEGACY_CONTRACT_ADDRESS=0xc68e401f19cc1fe1d4a703b87f96f164e3729b86
+VUE_APP_LEGACY_CONTRACT_FALLBACK_ADDRESSES=0xcC1eF3053681352E5B5F83ABEA72011825AfeD35
+VUE_APP_CLAIM_CONTRACT_ADDRESS=0xe24A7F24954a68B0f9B0862cFF83b817D9f63409
+VUE_APP_ACCEPTED_ISSUER_KEY=0xD354996e15E436c7c032A7be2d6218c38afEAc1a
+VUE_APP_CLAIM_FF=true
```

Comment:
This was probably commited by mistake, you can create a .env.local file with your local configurations which is ignored by git ;-)

Reply:
Ups, removed again